modules/mcuboot/hooks/nrf53_hooks.c

```diff
@@ -19,9 +19,51 @@
 #define NET_CORE_VIRTUAL_PRIMARY_SLOT 3
 
 #include <dfu/pcd.h>
+#include <fw_info_bare.h>
 
-int boot_read_image_header_hook(int img_index, int slot,
-		struct image_header *img_head)
+/**
+ * Compare image version numbers
```

Comment:
Should the docs be in some header file and in doxygen format?

Reply:
I'm not 100% set on the location of this function. I couldn't find a generic way to include this. Maybe I would start comparing it against he major version of the mcuboot header.

Reply:
there is no need to pull it out simply to be able to add doxygen docs to these IMO

Reply:
Sure will do.